content/github/administering-a-repository/renaming-a-branch.md

```diff
@@ -38,4 +38,10 @@ From the local clone of the repository on a computer, run the following commands
 $ git branch -m <em>OLD-BRANCH-NAME</em> <em>NEW-BRANCH-NAME</em>
 $ git fetch origin
 $ git branch -u origin/<em>NEW-BRANCH-NAME</em> <em>NEW-BRANCH-NAME</em>
+$ git remote set-head origin -a
+```
+
+Run the following command to remove tracking references to `OLD-BRANCH-NAME`:
```

Comment:
```suggestion
Optionally, run the following command to remove tracking references to the old branch name.
```

I'm making just a few small changes to your contribution.
- Since this is an optional step, we need to introduce it with "Optionally,"
- We should introduce steps (and lists) with full sentences, so I replaced the `:` with a `.`
- I think we can refer to the old branch name as "the old branch name" and it'll be a bit easier to understand the purpose of this step

I'm going to push these changes and get this merged 😄  Thank you for an awesome contribution!

Reply:
> Since this is an optional step, we need to introduce it with "Optionally,"
Ha! My original wording included "Optionally," and I removed it because I worry about sounding to passive 😁  Totally agree, thank you for explaining!

> We should introduce steps (and lists) with full sentences, so I replaced the `:` with a `.`
Ooo I like that format, using real sentences.  Cool 👍

> I think we can refer to the old branch name as "the old branch name" and it'll be a bit easier to understand the purpose of this step
Got it -- makes sense, yay for writing like humans not computers :)